add Button docs to site (#2285)
jpmorganchase/salt-ds: site/docs/components/button/usage.mdx

---
# Leave the frontmatter as is
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:

>   label: Usage

Reviewer: ```suggestion
```

### Using the Button component

- Keep the text description short, ideally one to three words and no more than five. If you’re including an icon with the text, use no more than three words.
- For icons, use the Salt Figma icon library. If the icon you need isn’t available, follow the steps in our [Creating Salt icons](<https://www.figma.com/file/ibGX0b0CxUcZU3pn6OrqX7/Salt%3A-Icons-(Community)?type=design&node-id=490-74&mode=design>) guide.

#### When to use Button

> - To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Cards](../cards) and [Forms](/salt/patterns/forms).

Reviewer 2: ```suggestion
- To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Card](../card) and [Form](/salt/patterns/form).
```

Author: Will update for card and keep the same for forms (https://www.saltdesignsystem.com/salt/patterns/forms)

> - To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Cards](../cards) and [Forms](/salt/patterns/forms).

Author: ```suggestion
- To allow the user to execute an action, such as “submit”, “merge” or “upload”. Buttons are typically placed in components and contexts such as [Dialog](../dialog), [Cards](../card) and [Forms](/salt/patterns/forms).
```
